docs/e2e-tests.md

```diff
@@ -64,19 +64,18 @@
 
 4. Deploy a cluster and run smoke test
 
-    Note that 'k8s-azure e2e' command will delete the cluster as last running step. To skip that, add parameter '-cskipcleanup=1' or set environment variable 'K8S_AZURE_SKIPCLEANUP=1'.
-
+    To deploy a cluster:
     ```
-    source <TestProfile>
-    CLUSTER_NAME=<ClusterName>
-    tests/k8s-azure/k8s-azure e2e -cname=$CLUSTER_NAME -cbuild_e2e_test=1 -caccm_image=<username>/azure-cloud-controller-manager:<image_version>
+    make deploy
     ```
 
     To connect the cluster:
     ```
-    source $CLUSTER_NAME/cluster.profile
-    kubectl version
+    cd tests/k8s-azure/manifest
```

Comment:
do we really need to cd?

Reply:
Yes sir actually we need to move to the file location else we need to change the deploy-cluster.sh a little more.